eth/config.go

```diff
@@ -121,6 +121,11 @@ type Config struct {
 
 	// Miscellaneous options
 	DocRoot string `toml:"-"`
+
+	// Type of the EWASM interpreter ("" for detault, or e.g. "evmc:path:option1:...")
```

Comment:
Does this needs to be colon separated or can anything be passed after `evmc:path:....` ?

Reply:
anything can be passed